server/routes/incentiveLevels.ts

```diff
@@ -61,6 +61,34 @@ export default function routes(router: Router): Router {
   router.get('/activate/:levelCode', requireRole, asyncMiddleware(activateDeactivate(true)))
   router.get('/deactivate/:levelCode', requireRole, asyncMiddleware(activateDeactivate(false)))
 
+  const moveLevel: { (move: 'up' | 'down'): RequestHandler } = move => async (req, res) => {
```

Comment:
May be worth adding a small comment to define "up" and "down" here and avoid any confusion.

As discussed briefly offline "up"/"down" on the list is a way of seeing this. There is also "up" as in "decrease sequence" 🙃. 

Although I appreciate:
- it's relative
- sequence/order is internal now
- UI is a placeholder and things will probably change

I think a comment doesn't hurt here :)

Reply:
we don't even know if we will need this route cos it is extremely ui dependant. but _fine_ ;)

Reply:
I'm not suggesting changing anything (as the current concept of up/down is **_not_** wrong).
But I'm thinking maybe add a small comment above here to explain what "up" and "down" _mean_ would avoid confusion :)